### ISSUE TITLE
take tasks from the DB

### ISSUE THREAD

**Issue by SimoravaIT** [SimoravaIT/Tamagotchi_Android#6]
the actual implementation use a list of tasks create in the code, need to be decided how to take the tasks and implement the function that create n Araylist with them

**Comment by edogab33** [comment]
> the actual implementation use a list of tasks create in the code

Can't see it in the code. Can you point it out?
However it's worth saying that hard coding _things_ is not an elegant implementation: a better solution is to model it in a local database, since in the real world if you would like to implement a backend server you can do it without modifying most of the architecture of the application. Furthermore it will be much easier to manage data between classes if there is a central database and finally you will need it anyway to keep the available tasks given in that day (unless you don't want to change them every time you open the app, which is pointless).

> need to be decided how to take the tasks and implement the function that create n Araylist with them

Read issue [#3](https://github.com/SimoravaIT/Tamagotchi_Android/issues/3).
Basically retrieving available tasks will be just an SQL query.
